backend/main.py

```
@@ -1,52 +1,49 @@
-from schemas import UserCreate, Token, LoginRequest, UserUpdate
-from utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email
+from schemas import UserCreate, Token, LoginRequest, UserUpdate, OrderOut, OrderCreate, PayBillRequest, ForgotPasswordRequest, ResetPasswordRequest
+from utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email, verify_reset_token, create_reset_token, send_reset_email
 from datetime import timedelta
 import os
 import dotenv
 from pydantic import BaseModel
 import uvicorn
-from typing import Annotated, List
-
+from typing import Annotated, List, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)